src/logic/Window.swift

```diff
@@ -101,7 +101,7 @@ class Window {
     }
 
     private func screenshot(_ bestResolution: Bool = false) -> NSImage? {
-        guard !isWindowlessApp, let cgWindowId = cgWindowId, cgWindowId != -1, let cgImage = cgWindowId.screenshot(bestResolution) else {
+        guard !isWindowlessApp, let cgWindowId = cgWindowId, cgWindowId != kCGNullWindowID, let cgImage = cgWindowId.screenshot(bestResolution) else {
```

Comment:
In theory, `CGWindowID` being unsigned, wid can never be negative. Thus comparing with `-1` was an actual bug, not a false positive error at compilation.

I'm wondering if comparing with `kCGNullWindowID` is ok. We set the wid to `0`, then get the actual wid from the OS API:

https://github.com/lwouis/alt-tab-macos/blob/012e3b5f3c321b249271420fffb3d7ee561b9560/src/api-wrappers/AXUIElement.swift#L54

Is there no way that the private API `_AXUIElementGetWindow` wouldn't set the ID, thus keeping it at `0`? We would not screenshot it then.

It's probably fine? What do you think?

Thank you